msi/release.sh

```diff
@@ -177,6 +177,9 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             <Component Directory="INSTALLFOLDER:\\usr\\share\\git\\">
                 <File Id="ReleaseNotes_Css" Source="$BUILD_EXTRA_WINPATH\\usr\\share\\git\\ReleaseNotes.css" />
             </Component>
+            <Component Directory="INSTALLFOLDER">
+                <RegistryValue Root="HKMU" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="\$(var.Version)" />

```

Comment:
Hehe, guess I should've wrote a better commit comment :wink:  The `\` is to escape the `$` since this is in a string inside of a bash script. `$` has a special meaning and needs to be escaped.